vivo/Dockerfile

```diff
@@ -32,14 +32,12 @@ RUN rm -r /usr/local/tomcat/webapps/examples
 
 # Set properties
 # Adjust logging, you can mount over this in docker-compose to further adjust
-COPY ./log4j.properties /usr/local/tomcat/webapps/VIVO/WEB-INF/classes
+COPY ./log4j.properties /usr/local/tomcat/webapps/vivo/WEB-INF/classes/log4j.properties
```

Comment:
The dockerfile copied ./log4j.properties to .../VIVO/WEB-INF/classes instead of .../vivo/WEB-INF/classes/log4j.properties  The previous dockerfile seems to write to the wrong filepath.